config/trackers.go

```diff
@@ -13,3 +13,8 @@ var Trackers = []string{
 	"udp://tracker.internetwarriors.net:1337/announce",
 	"http://mgtracker.org:6969/announce",
 	"http://tracker.baka-sub.cf/announce"}
+
+// NeededTrackers : Array indexes of Trackers for needed tracker in a torrent file
```

Comment:
When I read this: "Please include udp://tracker.doko.moe:6969 in your trackers.", I get that this is not optional but required. Or we should change it to: "You may add xxx"